server/graphql/v2/mutation/CreateCollectiveMutation.js

```diff
@@ -112,12 +114,58 @@ async function createCollective(_, args, req) {
           throw new ValidationFailed('Host account is not activated as Host.');
         }
       }
+
       const collective = await models.Collective.create(collectiveData, { transaction });
+
       // Add authenticated user as an admin
-      await collective.addUserWithRole(user, roles.ADMIN, { CreatedByUserId: user.id }, {}, transaction);
+      if (!args.skipDefaultAdmin) {
+        await collective.addUserWithRole(user, roles.ADMIN, { CreatedByUserId: user.id }, {}, transaction);
+      }
+
+      if (args.inviteMembers && args.inviteMembers.length) {
```

Comment:
I think we should add 2 restrictions there:
- Given that this can be used to know whether an email is registered on Open Collective, there should be a rate limit (to prevent email enumeration)
- `args.inviteMembers` should have a max length (30 admins?)

Reply:
In the current state, we're ultimately honoring `maxCoreContributorsPerAccount` because we're using `MemberInvitation.invite`, seems this is 60 admins.